mnemonics/src/main/java/org/kethereum/bip32/BIP32.kt

```diff
@@ -0,0 +1,18 @@
+package org.kethereum.bip32
+
+import org.kethereum.bip44.BIP44
+
+object BIP32 {
```

Comment:
I think we can remove this top level object - this is not java where we must do it - we have top level functions in Kotlin FTW ;-)

Reply:
done.